src/ConfirmProvider/ConfirmProvider.stories.js

```diff
@@ -0,0 +1,38 @@
+import { ConfirmProvider, useConfirm } from "./ConfirmProvider";
+
+import { Button } from "@mui/material";
+import React from "react";
+
+export default {
+  component: ConfirmProvider,
+  title: "General/ConfirmProvider"
+};
+
+function ConfirmDialog() {
+  const confirm = useConfirm();
+  const handleClick = () => {
+    confirm()
+      .then(() => {
+        console.log("confirmed");
```

Comment:
Use action so that these appear in the action log. 

```suggestion
        action("confirm");
```